service/telemetry.go

```diff
@@ -75,13 +74,20 @@ func (tel *telemetryInitializer) init(res *resource.Resource, settings servicete
 	}
 
 	settings.Logger.Info("Setting up own telemetry...")
+	configuredSDK, err := config.NewSDK(
```

Comment:
One question: are we not passing the resource anymore here?

Reply:
This is a good question. I'm working on an issue in the go-contrib repo where setting resource attributes via config is currently restricted to service name, as the go-jsonschema library doesn't have support for additionalProperties on an object that has other properties specified.

Putting this back into draft until that's resolved (https://github.com/open-telemetry/opentelemetry-go-contrib/issues/4828)

Reply:
@mx-psi here's roughly what i think the change will look like with a resource: https://github.com/open-telemetry/opentelemetry-collector/pull/9131/commits/51f6ed6a554182c23febabe2bb80827dfddf55e1

the code isn't in the config package yet though, so it will likely change

Reply:
:+1: Makes sense, let's wait on that issue to be solved then :)